apps/doji-frontend/src/pages/session-v2/[id]/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { httpClient } from '@frontend/services'
+import { ExtendedNextPage } from '@frontend/type'
+import { IPublicSessionReviewDTO, IReviewStat, ISessionInformationDTO } from '@libs/api'
+import {
+  Avatar,
+  Button,
+  CircularProgress,
+  Grid,
+  LinearProgress,
+  Link,
+  Stack,
+  TextField,
+  Typography,
+  useTheme,
+} from '@mui/material'
+import { Box } from '@mui/system'
+import { formatDistanceToNow } from 'date-fns'
+import { useRouter } from 'next/router'
+
+import React from 'react'
+import { AiFillStar, AiOutlineStar } from 'react-icons/ai'
+import { BsThreeDotsVertical } from 'react-icons/bs'
+import { useQuery } from 'react-query'
+
+function SessionDetail() {
+  return (
+    <Box>
```

Comment:
```suggestion
    <div>
```

Box with no props = div